cmd/ztest/ztest.c

```diff
@@ -123,7 +123,7 @@
 #include <math.h>
 #include <sys/fs/zfs.h>
 #include <libnvpair.h>
-#ifdef __GNUC__
+#ifdef __GLIBC__
```

Comment:
Seems reasonable.  But can you add a link to the commit comment explaining why __GLIBC__ should be used instead of __GNUC__.  My guess is that __GNUC__ was used here since that's what recommenced fron by gcc documentation.

https://gcc.gnu.org/onlinedocs/gcc/C-Extensions.html